src/libsystemd-network/test-dhcp6-client.c

```diff
@@ -1010,6 +1081,7 @@ TEST(dhcp6_client) {
         assert_se(sd_dhcp6_client_set_local_address(client, &local_address) >= 0);
         assert_se(sd_dhcp6_client_set_fqdn(client, "host.lab.intra") >= 0);
         assert_se(sd_dhcp6_client_set_iaid(client, unaligned_read_be32((uint8_t[]) { IA_ID_BYTES })) >= 0);
+        assert_se(sd_dhcp6_client_set_send_release(client, 1) == 0);
```

Comment:
This is a test code, hence not a big matter, but please use `true` even the argument type is `int`.